client-templates/nexus/.env.sample

```diff
@@ -18,12 +18,17 @@ ACCEPT=accept.json
 # The path for the broker's internal healthcheck URL. Must start with a '/'.
 BROKER_HEALTHCHECK_PATH=/healthcheck
 
-# Provide RES_BODY_URL_SUB with the URL of the Nexus without credentials and http protocol
+# Provide RES_BODY_URL_SUB with the URL of the Nexus without credentials
 # This URL substitution is required for NPM integration
-# RES_BODY_URL_SUB=http://<your.nexus.hostname>/repository
+# RES_BODY_URL_SUB=https://<your.nexus.hostname>/repository
```

Comment:
should this be uncommented?

Reply:
it's only required for npm/yarn - best to let the customer decide I think